### ISSUE TITLE
Remove `.guestfs-1000` directory before creating the bundle tarball

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#106]
As of now after extracting the bundle and checking the content looks like it have `.guestfs-1000` hidden directory which should not be part of end bundle.

```
$ ls -al ~/.crc/cache/crc_libvirt_4.2.0-0.nightly-2019-09-26-192831/
total 11365608
drwxrwxr-x. 3 prkumar prkumar        4096 Sep 29 18:38 .
drwxrwxr-x. 6 prkumar prkumar        4096 Sep 29 18:34 ..
-rw-rw-r--. 1 prkumar prkumar        1133 Sep 29 18:38 crc-bundle-info.json
-rw-r--r--. 1 prkumar prkumar 11638341632 Sep 29 18:38 crc.qcow2
drwxr-xr-x. 3 prkumar prkumar        4096 Sep 29 18:34 .guestfs-1000
-r--------. 1 prkumar prkumar        1811 Sep 29 18:38 id_rsa_crc
-rw-r--r--. 1 prkumar prkumar          23 Sep 29 18:38 kubeadmin-password
-rw-r--r--. 1 prkumar prkumar        8939 Sep 29 18:38 kubeconfig

$ du -sh ~/.crc/cache/crc_libvirt_4.2.0-0.nightly-2019-09-26-192831/.guestfs-1000/appliance.d/
4.1G	/home/prkumar/.crc/cache/crc_libvirt_4.2.0-0.nightly-2019-09-26-192831/.guestfs-1000/appliance.d/
```

**Comment by cfergeau** [comment]
Yes, this used to be removed before the code was deleted and then partially readded
https://github.com/code-ready/snc/commit/a76e8721362f9295
An alternative may be to set TMPDIR= to $(pwd) rather than $(pwd)/$destDir

**Comment by praveenkumar** [comment]
Fixed by #107
